interfaces/ISO15118_charger.yaml

```diff
@@ -327,61 +327,19 @@ vars:
   dlink_pause:
     description: Request power saving mode, while staying MATCHED.
     type: "null"
-  EV_AppProtocol:
+  ev_app_protocol:
     description: >-
       Debug_Lite - This request message provides a list of charging protocols
       supported by the EVCC
-    type: array
-    items:
-      description: >-
-        This message element is used by the EVCC for transmitting the list
-        of supported protocols
-      type: object
-      properties:
-        ProtocolNamespace:
-          description: >-
-            This message element is used by the EVCC to uniquely identify
-            the Namespace URI of a specific protocol supported by the EVCC
-          type: string
-          minLength: 1
-          maxLength: 100
-        VersionNumberMajor:
-          description: >-
-            This message element is used by the EVCC to indicate the major
-            version number of the protocol
-          type: integer
-          minimum: 0
-        VersionNumberMinor:
-          description: >-
-            This message element is used by the EVCC to indicate the minor
-            version number of the protocol
-          type: integer
-          minimum: 0
-        SchemaID:
-          description: >-
-            This message element is used by the EVCC to indicate the SchemaID
-            assigned by the EVCC to the protocol
-          type: integer
-          minimum: 0
-          maximum: 255
-        Priority:
-          description: >-
-            This message element is used by the EVCC for indicating the
-            protocol priority of a specific protocol allowing the SECC to select a
-            protocol based on priorities
-          type: integer
-          minimum: 1
-          maximum: 20
-      additionalProperties: false
-    minItems: 1
-    maxItems: 20
-  V2G_Messages:
+    type: object
+    $ref: /iso15118_charger#/AppProtocols
+  v2g_messages:
     description: >-
       Debug - This element contains all V2G elements and should be used
       for debug purposes only
     type: object
-    $ref: /iso15118_charger#/V2G_Messages
-  Selected_Protocol:
+    $ref: /iso15118_charger#/V2gMessages
```

Comment:
rather `V2GMessages` ?

Reply:
V2gMessages looks weird but technically it is correct pascal case

Reply:
I would prefer V2GMessages, but V2gMessages is correct here